src/main/kotlin/com/atlassian/performance/tools/report/api/judge/PercentageExtensions.kt

```diff
@@ -0,0 +1,4 @@
+package com.atlassian.performance.tools.report.api.judge
+
+fun Float.toPercentage(decimalPlaces: Int): String = "%+.${decimalPlaces}f%%".format(this * 100)
```

Comment:
- [ ] move it out of `api` package (or make it private)

Otherwise it leaks as API we'd have to maintain